src/test/java/org/kainos/groupA/integration/UserIntegrationTest.java

```diff
@@ -0,0 +1,61 @@
+package org.kainos.groupA.integration;
+
+import io.dropwizard.configuration.ResourceConfigurationSourceProvider;
+import io.dropwizard.testing.junit5.DropwizardAppExtension;
+import io.dropwizard.testing.junit5.DropwizardExtensionsSupport;
+import org.eclipse.jetty.http.HttpStatus;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.kainos.groupA.GroupAApplication;
+import org.kainos.groupA.GroupAConfiguration;
+import org.kainos.groupA.models.User;
+
+import javax.ws.rs.client.Entity;
+import javax.ws.rs.core.MediaType;
+import javax.ws.rs.core.Response;
+
+import java.nio.charset.Charset;
+import java.util.Random;
+
+import static org.hibernate.validator.internal.util.Contracts.assertTrue;
+
+@ExtendWith(DropwizardExtensionsSupport.class)
+public class UserIntegrationTest {
+    User testUser =  new User(
+            "test@kainos.com",
+            "$2b$10$ktJIlwDzSgj/kCgu3WaX/Or00YlPkye77zUSsDJtGZy/atH4c9xcK",
+            User.Role.Employee,
+            "Joe",
+            "Doe",
+            "123456789",
+            User.Location.Birmingham
+    );
+    static final DropwizardAppExtension<GroupAConfiguration> APP = new DropwizardAppExtension<GroupAConfiguration>(
+            GroupAApplication.class, null,
+            new ResourceConfigurationSourceProvider()
+    );
+
+    private String generateEmail() {
+        String CHARS = "ABCDEFGHIJKLMNOPQRSTUVWXYZ1234567890";
+        StringBuilder salt = new StringBuilder();
+        Random rnd = new Random();
+        while (salt.length() < 10) {
+            int i = (int) (rnd.nextFloat() * CHARS.length());
+            salt.append(CHARS.charAt(i));
+        }
+        return salt.toString() + "@kainos.com";
+    }
+
+    @Test
+    void createUser_shouldReturnCreatedUserId() {
```

Comment:
can you add a test for 400 response code as well? I'm not sure if you can force the 500 error in an integration test but if you can find a way then add one for that also